jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/messages_uk.properties

```diff
@@ -1,50 +1,50 @@
-label.answer.voteUp.tooltip=\u0412\u0456\u0434\u043F\u043E\u0432\u0456\u0434\u044C \u043A\u043E\u0440\u0438\u0441\u043D\u0430
-label.answer.voteDown.tooltip=\u0412\u0456\u0434\u043F\u043E\u0432\u0456\u0434\u044C \u043D\u0435 \u043A\u043E\u0440\u0438\u0441\u043D\u0430, \u043E\u0431\u0440\u0430\u0437\u043B\u0438\u0432\u0430, \u043F\u0440\u043E\u043F\u043E\u043D\u0443\u0454 \u043F\u043E\u0441\u0438\u043B\u0430\u043D\u043D\u044F \u043D\u0430 \u0437\u043E\u0432\u043D\u0456\u0448\u043D\u0456 \u0440\u0435\u0441\u0443\u0440\u0441\u0438 \u0431\u0435\u0437 \u0434\u043E\u0434\u0430\u0442\u043A\u043E\u0432\u0438\u0445 \u043F\u043E\u044F\u0441\u043D\u0435\u043D\u044C
-label.confirm.self.answer.title=\u0412\u0438 \u0442\u043E\u0447\u043D\u043E \u0431\u0430\u0436\u0430\u0454\u0442\u0435 \u0432\u0456\u0434\u043F\u043E\u0432\u0456\u0441\u0442\u0438 \u043D\u0430 \u0441\u0432\u043E\u0454 \u043F\u0438\u0442\u0430\u043D\u043D\u044F?
-label.confirm.self.answer.message=\u041A\u043E\u0440\u0438\u0441\u0442\u0443\u0439\u0442\u0435\u0441\u044F \u043A\u043E\u043C\u0435\u043D\u0442\u0430\u0440\u044F\u043C\u0438, \u044F\u043A\u0449\u043E \u0431\u0430\u0436\u0430\u0454\u0442\u0435 \u043F\u0440\u043E\u043A\u043E\u043C\u0435\u043D\u0442\u0443\u0432\u0430\u0442\u0438 \u0447\u0438\u044E\u0441\u044C \u0432\u0456\u0434\u043F\u043E\u0432\u0456\u0434\u044C.<br>\u0410\u0431\u043E \u0432i\u0434\u0440\u0435\u0434\u0430\u0433\u0443\u0439\u0442\u0435 \u0441\u0432\u043E\u0454 \u043F\u0438\u0442\u0430\u043D\u043D\u044F, \u044F\u043A\u0449\u043E \u0445\u043E\u0447\u0435\u0442\u0435 \u0434\u043E\u0434\u0430\u0442\u0438 \u0431\u0456\u043B\u044C\u0448\u0435 \u0456\u043D\u0444\u043E\u0440\u043C\u0430\u0446\u0456\u0457.
-label.confirm.multi.answer.title=\u0412\u0438 \u0442\u043E\u0447\u043D\u043E \u0431\u0430\u0436\u0430\u0454\u0442\u0435 \u0432\u0456\u0434\u043F\u043E\u0432\u0456\u0441\u0442\u0438 \u043D\u0430 \u043F\u0438\u0442\u0430\u043D\u043D\u044F \u0449\u0435 \u0440\u0430\u0437?
-label.confirm.multi.answer.message=\u041A\u043E\u0440\u0438\u0441\u0442\u0443\u0439\u0442\u0435\u0441\u044F \u043A\u043E\u043C\u0435\u043D\u0442\u0430\u0440\u044F\u043C\u0438, \u044F\u043A\u0449\u043E \u0431\u0430\u0436\u0430\u0454\u0442\u0435 \u043F\u0440\u043E\u043A\u043E\u043C\u0435\u043D\u0442\u0443\u0432\u0430\u0442\u0438 \u0447\u0438\u044E\u0441\u044C \u0432\u0456\u0434\u043F\u043E\u0432\u0456\u0434\u044C.<br>\u0410\u0431\u043E \u0432\u0456\u0434\u0440\u0435\u0434\u0430\u0433\u0443\u0439\u0442\u0435 \u0441\u0432\u043E\u044E \u043F\u0435\u0440\u0448\u0443 \u0432\u0456\u0434\u043F\u043E\u0432\u0456\u0434\u044C, \u044F\u043A\u0449\u043E \u0445\u043E\u0447\u0435\u0442\u0435 \u0434\u043E\u0434\u0430\u0442\u0438 \u0431\u0456\u043B\u044C\u0448\u0435 \u0456\u043D\u0444\u043E\u0440\u043C\u0430\u0446\u0456\u0457.
+label.addQuestion=Задати Питання
```

Comment:
These changes are incorrect - the content should be kept in ansi format (with `\uxxxx` for unicode symbols), not as a unicode text.